src/dialects/sqlite/query-generator.js

```diff
@@ -366,18 +366,10 @@ export class SqliteQueryGenerator extends SqliteQueryGeneratorTypeScript {
     return result;
   }
 
-  showConstraintsQuery(tableName, constraintName) {
-    let sql = `SELECT sql FROM sqlite_master WHERE tbl_name='${tableName}'`;
-
-    if (constraintName) {
-      sql += ` AND sql LIKE '%${constraintName}%'`;
```

Comment:
Why was this removed? Did it not work as expected?

Reply:
I can add this back in however it's passing a name is not going to filter the results to the `constraintName` value, all constraints will be returned for the table unless the value of `constraintName` doesn't exist in which case an empty array would be returned.

It might be better to throw an error if a value is passed informing the user that SQLite doesn't support filtering constraints.